aospy/test/aospy_test_objs/aospy_user/regions/__init__.py

```diff
@@ -0,0 +1,355 @@
+from aospy.region import Region
```

Comment:
We should have only a few regions.  One global, one that's a simple lat-lon box, one that has multiple lat-lon bounds, one with land mask, and one without land mask.